src/setup/rgw_service

```diff
@@ -14,10 +14,12 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+index=$1
+
 function start_rgw {
   machine_id=$(cat /etc/machine-id)
   # TODO: To use log dir from config
-  exec sudo /usr/bin/radosgw -f --name client.rgw -c /etc/cortx/rgw/"$machine_id"/cortx_rgw.conf --no-mon-config &> /var/log/ceph/rgw.log &
+  exec sudo /usr/bin/radosgw -f --name client.rgw-"$index" -c /etc/cortx/rgw/"$machine_id"/cortx_rgw.conf --no-mon-config &> /var/log/ceph/rgw.log &
```

Comment:
We should not use hard coded path. This should the config path

Reply:
Removed hardcoded path.